app/grants/templates/grants/components/card.html

```diff
@@ -0,0 +1,229 @@
+        <i class="fa fa-stop" aria-hidden="true"></i> Cannot Fund your Own Grant
+      </button>
+    </div>
+  </div>
+  <div class="grant-item" v-else>
+    <a :href="grant.details_url">
+    <div class="grant-item__img">
+      <img :src="grant.logo_url">
+    </div>
+    <div class="grant-item__content px-3">
+      <h2 class="grant-item__title font-subheader">
+        <a :href="grant.details_url">[[ grant.title.slice(0,60) ]][[ grant.title.length > 60 ? '...' : '']]</a>
+      </h2>
+      <p class="grant-item__pitch font-caption mb-2">[[ grant.description.slice(0, 145) ]][[ grant.description.length > 145 ? '...' : '']]</p>
+      <p class="grant-item__pitch font-caption mb-0" style="height:2rem;"  data-toggle="tooltip" data-placement="top" title="The last time the grant admin updated the grant." >
+        <i style="width: 14px;" class="fas fa-clock mt-2 mb-0 text-center"></i>
+        Last Update: [[ grant.last_update ?  grant.last_update_natural : 'Before Round 6' ]]
+
+        <template v-if="cred.is_staff">
+          <hr>
+          <i class="fa fa-lock" aria-hidden="true"></i>
+          SybilScore ™️: [[ grant.sybil_score ]]
+          <br>
+          <i class="fa fa-lock" aria-hidden="true"></i>
+          RiskScore ™️: [[ grant.weighted_risk_score ]]
+        </template>
+
+      </p>
+
+      <div class="grant-item__owner justify-content-between">
+        <div>
+          <span class="grant-item__owner-handle font-body">{% trans 'by' %}</span>
+          <a :href="grant.admin_profile.url" :data-usercard="grant.admin_profile.handle" target="_blank" rel="noopener noreferrer">
+            <div class="grant-item__owner mt-3 mb-4">
+              <div class="grant-item__owner-image ml-2">
+                <img :src="grant.admin_profile.avatar_url || 'https://c.gitcoin.co/avatars/57e79c0ae763bb27095f6b265a1a8bf3/thelostone-mc.svg'">
+              </div>
+              <div class="grant-item__owner-handle font-body">
+                <span class="ml-2">
+                  [[ grant.admin_profile.handle || 'anonymous' ]]
+                </span>
+              </div>
+            </div>
+          </a>
+        </div>
+          <template v-if="cred.is_authenticated">
+            <button :class="`text-decoration-none btn btn-link star-action m-0 p-0 mr-4 mt-3 gc-text-blue ${!grant.favorite ? 'text-muted' : ''}`" :data-grant="grant.id">
+              <i :class="`${!grant.favorite ? 'far' : 'fa' } fa-star`" ></i>
+              <span class="ml-1 font-body">[[ !grant.favorite ? 'Follow' : 'Following' ]]</span>
+            </button>
+          </template>
+        </div>
+    </div>
+    <div class="grant-item__footer">
+      <template v-if="1">
+          <div class="clr mb-3">
+            <div class="match__round pb-1 pt-2 mb-2">
+              <h2 class="title col-8 font-caption g-bold">
+                  CLR MATCH ROUND 6
+              </h2>
```

Comment:
^ why is this hardcoded ?
This is something which should be available from the backend API.